config-test/test_server_details.py

```diff
@@ -11,10 +12,9 @@ def conf():
 
 
 def test_version(conf, env):
-    response = requests.get(conf.get(env, 'reader_server') + '/__version__')
+    response = requests.get(conf.get(env, 'reader_server') + '__version__')
     data = response.json()
-
-    expected_fields = {'version', 'source', 'name', 'build', 'commit'}
+    expected_fields = conf.get(env, 'version_fields').split(', ')
```

Comment:
Maybe we can build a `aslist` helper that can help us improve this in the future.

See http://docs.pylonsproject.org/projects/pyramid/en/latest/_modules/pyramid/settings.html#aslist for instance.
We can replace spaces by ','